hivemind/p2p/p2p_daemon.py

```diff
@@ -91,6 +91,7 @@ async def create(
         use_auto_relay: bool = False,
         relay_hop_limit: int = 0,
         startup_timeout: float = 15,
+        idle_timeout: int = 0,
```

Comment:
```suggestion
        idle_timeout: float = 0,
```

Usually, timeouts are `float` (e.g. this is useful to specify the timeout such that `0 < idle_timeout < 1`). Can we implement it here as well?